### PR TITLE
PascalCase name for tuple members

### PR DESCRIPTION
**Pull request by TKharaishvili**
Barely a contribution

### CODE REVIEW

File: src/Minsk/CodeAnalysis/Emit/Emitter.cs [hunk 1]
```diff
@@ -57,7 +57,7 @@ private Emitter(string moduleName, string[] references)
                 }
             }
 
-            var builtInTypes = new List<(TypeSymbol type, string MetadataName)>()
+            var builtInTypes = new List<(TypeSymbol type, string metadataName)>()
```
Comment (ostorc) [inline src/Minsk/CodeAnalysis/Emit/Emitter.cs:60]: If you want to change this, it should be the other way around. As you can see in this [comment](https://github.com/dotnet/runtime/issues/27939#issuecomment-531420515), the .NET Runtime team decided to use the Pascal naming convention (also the summary was written by the Minsk master @terrajobst, so another reason why use this convention  :) )

Comment (ltrzesniewski) [reply src/Minsk/CodeAnalysis/Emit/Emitter.cs:60]: That's the convention for the BCL, which, let's be honest, is an obvious mistake. 😉 
We don't need to repeat that mistake here.

In all seriousness, most of the tuples you'll see in the wild seem to use camelCase. The BCL convention is not really followed that much.

Comment (TKharaishvili) [reply src/Minsk/CodeAnalysis/Emit/Emitter.cs:60]: @ostorc @ltrzesniewski 
There's another place in the codebase where pascal case is used - https://github.com/terrajobst/minsk/blob/7223f8249c0693ab1cbe080b66681a5178c96467/src/Minsk/CodeAnalysis/Emit/Emitter.cs#L38
(I discovered it after opening this PR)

Should we ask Immo for the executive decision? @terrajobst what do you say?

Comment (ostorc) [reply src/Minsk/CodeAnalysis/Emit/Emitter.cs:60]: @ltrzesniewski for me it feels quite unnatural to access tuple member which was declared with camel case. It just doesn't feel right. If I want to be it in camel case I deconstruct the tuple into local variables.

Comment (ostorc) [reply src/Minsk/CodeAnalysis/Emit/Emitter.cs:60]: But again, this discussion was already held in the runtime issue. We should probably just await the response from @terrajobst

Comment (terrajobst) [reply src/Minsk/CodeAnalysis/Emit/Emitter.cs:60]: We decided to go with PascalCase for one reason: usage. When accessing tuple members, they should behave like any other public field/property, which is PascalCase. This also makes it easier to refactor the code base from tuples to custom types. So yeah, PascalCasing is the correct casing for tuples. I consider any deviation from this in Minks to be a bug :-)

Comment (TKharaishvili) [reply src/Minsk/CodeAnalysis/Emit/Emitter.cs]: Pushed a new commit reflecting the final decision.

**Pull request merged**